src/cli/java/org/commcare/util/engine/CommCareConfigEngine.java

```diff
@@ -214,9 +214,11 @@ private void init(String profileRef) {
         }
     }
 
-    public void installAppFromReference(String profileReference) throws UnresolvedResourceException,
+    public void installAppFromReference(String profileReference)
+            throws UnresolvedResourceException,
             UnfullfilledRequirementsException, InstallCancelledException {
-        ResourceManager.installAppResources(platform, profileReference, this.table, true);
+        ResourceManager.installAppResources(platform, profileReference, this.table, true,
+                Resource.RESOURCE_AUTHORITY_LOCAL);
```

Comment:
Was surprised this was safe but appears to be - I guess the CLI never supported installing from the profile reference?